flow/embedded_views.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+//
+#ifndef FLUTTER_FLOW_EMBEDDED_VIEWS_H_
+#define FLUTTER_FLOW_EMBEDDED_VIEWS_H_
+
+#include "flutter/fml/memory/ref_counted.h"
+
+namespace flow {
+
+class EmbeddedViewCompositingParams {
+ public:
+  double translateXPixels;
```

Comment:
```
SkPoint offset;
SkSize bounds;
```

Is more idiomatic engine code.

Reply:
Renamed to offsetPixels, but kept sizePoints. (I want to emphasize the difference between pixels and points here)